loom/script/native/core/system/lmJSON.cpp

```diff
@@ -645,5 +723,5 @@ static int registerSystemJSON(lua_State *L)
 
 void installSystemJSON()
 {
-    NativeInterface::registerNativeType<JSON>(registerSystemJSON);
+    NativeInterface::registerManagedNativeType<JSON>(registerSystemJSON);
```

Comment:
I think that some of the LoomScript-only functions I added necessitated the switch to a managed type.